launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml

```diff
@@ -1,30 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
   <arg name="output_pose_with_cov_name" default="/localization/pose_estimator/pose_with_covariance"/>
-  <node pkg="eagleye_fix2pose" name="fix2pose_node" exec="fix2pose">
-    <!-- /eagleye/fix or  /GNSS/fix(GNSS pose only mode)-->
-    <remap from="eagleye/fix" to="eagleye/fix"/>
+
+  <node pkg="eagleye_geo_pose_converter" name="geo_pose_converter_node" exec="geo_pose_converter" output="screen">
     <remap from="eagleye/pose_with_covariance" to="$(var output_pose_with_cov_name)"/>
+
     <!-- plane rectangular coordinate number -->
     <param name="plane" value="7"/>
     <!-- 1 : plane rectangular coordinate  2 : MGRS -->
     <param name="tf_num" value="2"/>
     <!-- 0 : No convert  1 : ellipsoid -> altitude  2 : altitude -> ellipsoid -->
     <param name="convert_height_num" value="0"/>
-    <!-- 0 : EGM2008-1  1 : Japanese Geoid 2011 Ver2.1 -->
+    <!-- cspell:ignore GSIGE2011 -->
+    <!-- 0 : EGM2008-1  1 : GSIGE2011 Ver2.1 -->
```

Comment:
Do you mean `GSIGEO2011`?
[related link](https://www.gsi.go.jp/ENGLISH/Bulletin62_CONTENTS.html#62-2:~:text=Development%20of%20new%20hybrid%20geoid%20model%20for%20Japan%2C%20%22GSIGEO2011%22)

Reply:
@rsasaki0109  FYI: eagleye has also have an inccorect notation?
https://github.com/MapIV/eagleye/blob/30bae75dace6d9bfed0d67d51d6eec5928932509/eagleye_util/fix2pose/launch/fix2pose.xml#L15

Reply:
Thank you for pointing it out. It's a misspelling.

Reply:
Thanks, corrected: https://github.com/autowarefoundation/autoware.universe/pull/5506/commits/d64b8feed171573d63cf9b255028d453aef18d7e